app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,6 @@
 <resources>
     <string name="app_name">Art Institute of Chicago</string>
+
+    <string name="google_app_id">1:493215409123:android:3aa040c3145f1213</string>
```

Comment:
Should this be here? Can we use the gradle `resValue` method instead?

    resValue "string", "google_app_id", "whatever"